doc/subscribe.rst

```diff
@@ -7,11 +7,12 @@ Subscribe
 Dipy_ is a part of the Nipy_ community and we are happy to share the same
 e-mail list. This makes sense as we can share ideas with a broader
 community and boost collaboration across neuroimagers. If you
-want to post to the list you will need first to subscribe to `nipy
+want to post to the list you will need first to subscribe to the `nipy
 mailing list`_. We suggest to begin the subject of your e-mail with
 ``[Dipy]``. This will help us respond faster to your questions.
 
-Best wishes,
-Dipy developers
+Additional help can be found on the Neurostars_ website. You can 
+sometimes catch us on the `dipy gitter`_ channel.
```

Comment:
I would still simply write ``Additional help can be found on the Neurostars website or in `dipy gitter` `` Also, I think it makes sense to send an e-mail around and ask if the other dipy developers want to have gitter in the picture for user support. From what I see from other projects, gitter is used mostly as a forum for developers and github users but not for every user. Everyone of course should be able to get in or read the discussions but I am not sure that this is the correct framework for Q&A or user support. We may need to open a new channel only for that if we want to have a forum based support. I suppose you are assuming in all this that the traffic in gitter will stay low for some time. But you never know... Anyway no strong opinions here on my side. Your call.